etcdserver/raft.go

```diff
@@ -198,6 +201,13 @@ func (r *raftNode) start(s *EtcdServer) {
 					return
 				}
 
+				// the leader can write to its disk in parallel with replicating to the followers and them
```

Comment:
`...can write to its disk while concurrently replicating to the followers, and then followers can write to their disks`?